layouts/Footer.tsx

```diff
@@ -0,0 +1,165 @@
+    },
+    {
+      id: 3,
+      content: "BLOG",
+    },
+    {
+      id: 4,
+      content: "BLOG POST",
+    },
+    {
+      id: 5,
+      content: "SHOP",
+    },
+    {
+      id: 6,
+      content: "SHOP SINGLE",
+    },
+  ];
+  const utilityPages = [
+    {
+      id: 1,
+      content: "404 ERROR PAGE",
+    },
+    {
+      id: 2,
+      content: "PASSWORD PROTECTED",
+    },
+    {
+      id: 3,
+      content: "STYLEGUIDE",
+    },
+    {
+      id: 4,
+      content: "LICENSING",
+    },
+    {
+      id: 5,
+      content: "CHANGELOG",
+    },
+  ];
+  return (
+    <footer className={styles["footer__container"]}>
+      <div className={styles["footer__main__container"]}>
+        <div className={styles["footer__main__fylla__container"]}>
+          <div className={styles["footer__main__fylla__top__container"]}>
+            <div className={styles["footer__main__fylla__top__icon__container"]}>
+              <Image
+                src={fylla}
+                alt="icon"
+                className={styles["footer__main__fylla__top__icon"]}
+                aria-label="icon__fylla"
+              />
+              <p className={`${styles["footer__main__fylla__top__title"]} ${titleFont.className}`}>
+                TAOTECH
+              </p>
+            </div>
+            <p className={`${textFont.className} global__text-default`}>
+              Main Street. 1<br /> 22222 Berlin
+            </p>
+          </div>
+          <ul className={styles["footer__main__fylla__ul"]}>
+            <li>
+              <Image src={badgeLN} alt="badge" aria-label="badge__ln" />
+            </li>
+            <li>
+              <Image src={badgeIN} alt="badge" aria-label="badge__in" />
+            </li>
+            <li>
+              <Image src={badgeFB} alt="badge" aria-label="badge__fb" />
+            </li>
+          </ul>
+        </div>
+        <div className={styles["footer__main__links__container"]}>
+          <div className={styles["footer__main__link__container"]}>
+            <ul
+              className={`${styles["footer__main__link__ul"]} ${titleFont.className} global__text-meta`}
+            >
+              <li className={`global__heading-h4 ${titleFont.className}`}>Pages</li>
+              {pages.map((page) => (
+                <FooterNaviLink key={page.id} content={page.content} />
+              ))}
+              <li>
+                <Button
+                  buttonWidth="172px"
+                  buttonHeight="35px"
+                  isBlack={true}
+                  placeHolder="More Templates"
+                ></Button>
+              </li>
+            </ul>
+          </div>
+          <div className={styles["footer__main__link__container"]}>
+            <ul
+              className={`${styles["footer__main__link__ul"]} ${titleFont.className} global__text-meta`}
+            >
+              <li className={`global__heading-h4 ${titleFont.className}`}>CMS</li>
+              {cms.map((c) => (
```

Comment:
The letter c is not an appropriate naming for this item.

Reply:
I have changed 'c' to 'cmsItem'